packages/core/src/@types/txbuilder.ts

```diff
@@ -22,15 +22,23 @@ export interface ITxBuilderComplete {
   submit: () => Promise<string>;
 }
 
+/**
+ * The referral fee object if set.
+ */
+export interface ITxBuilderReferralFee {
+  destination: string;
+  payment: AssetAmount<IAssetAmountMetadata>;
```

Comment:
Do we already configure the `quantity` here as well? Is the referral fee hardcoded or dynamic based on the tx?

Reply:
The `quantity` is in the `payment` since that's an `AssetAmount` class. The referral fee is set globally when you create the Transaction Builder, here: https://github.com/SundaeSwap-finance/sundae-sdk/pull/18/files#diff-d7a6ea4693a4fad15087689d3f007e9bc929e0f24f15f97c90d857a9029970e6R39-R55